eu.retroshine.RetroShine.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: eu.retroshine.RetroShine
+runtime: org.kde.Platform
+runtime-version: "6.9"
+sdk: org.kde.Sdk
+command: RetroShine
+finish-args:
+  - --share=network
+  - --filesystem=home  # linter error: comment on PR
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --persist=RetroShine
+
+modules:
+  - name: retroshine-binary
+    buildsystem: simple
+    build-commands:
+      - mkdir ${FLATPAK_DEST}/bin
+      # this fails: folder "bin" does not exist!
+      - ln -s ${FLATPAK_DEST}/extra/RetroShine/RetroShine ${FLATPAK_DEST}/bin/RetroShine
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+    sources:
+      - type: extra-data
+        filename: RetroShine-1.1.1-x86_64.tar.gz
+        only-arches:
+          - x86_64
+        # Problem: "url" may cause connection error, "path" is not supported by type "extra-data", only by type "file".
+        #url: https://retroshine.eu/download/RetroShine-1.1.1-x86_64.tar.gz
+        #url: https://drive.usercontent.google.com/download?id=1sbMpnBxTvT8uHWpdxaJ580iGsqG4CbBZ&export=download&confirm
+        url: https://github.com/yellowdolphin/retroshine/releases/download/v1.1.1/RetroShine-1.1.1-x86_64.tar.gz
+        sha256: c6b9126814d97eea7d4acfc924f28310594b28687e62b7921bddbad3c31dde82
+        size: 222648883
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          # where does the downloaded files go? -> /app/extra/
+          # this is current dir
+          # but FLATPAK_DEST is not set during install (only build)
+          # install with option -v to see output
+          - tar xfz RetroShine-1.1.1-x86_64.tar.gz
+          - ls -l RetroShine/RetroShine
+          # use softlink instead of this:
+          #- cp -r RetroShine/* /app/bin/
+          #- chmod +x /app/bin/RetroShine
+
+  - name: metainfo
```

Comment:
this doesn't need to be a separate module.

Reply:
done